packages/server/src/api/muster/muster.controller.it.ts

```diff
@@ -50,12 +50,17 @@ describe(`Muster Controller`, () => {
     });
 
     it(`should return full compliance with a single active config`, async () => {
-      const res = await req.get(`/${org1}/${unit1}/complianceByDateRange?isoStartDate=2020-01-03&isoEndDate=2020-01-04`);
+      const res = await req.get(`/${org1}/${unit5}/complianceByDateRange?isoStartDate=2020-01-03&isoEndDate=2020-01-04`);
       expectNoErrors(res);
 
       // The single config is active on jan 3 so we select the item in the list for testing
       const dayRate = res.data.musterComplianceRates.find((rate: GetMusterComplianceByDateResponse) => rate.isoDate.split('T')[0] === '2020-01-03');
       expect(dayRate.musterComplianceRate).equal(1);
     });
+
+    it(`should error with bad date input`, async () => {
+      const res = await req.get(`/${org1}/${unit5}/complianceByDateRange?isoStartDate=YYYY-MM-DD&isoEndDate=2020-01-04`);
```

Comment:
Would the test fail for an invalid date? For example: 2020-99-03
I think that might be a better test then using `YYYY-MM-DD` as a value.
First I thought we are sending a date format, then I realized what is being tested here.

Reply:
Test will still fail but I can add that value case

Reply:
Yep, I added a numeric value as part of the assertions for that test now too.